src/loaders/OmeZarrLoader.ts

```diff
@@ -360,23 +450,23 @@ class OMEZarrLoader extends ThreadableVolumeLoader {
     onData: RawChannelDataCallback
   ): Promise<{ imageInfo: ImageInfo }> {
     const maxExtent = this.maxExtent ?? new Box3(new Vector3(0, 0, 0), new Vector3(1, 1, 1));
-    const [z, y, x] = this.axesTCZYX.slice(2);
+    const [z, y, x] = this.sources[0].axesTCZYX.slice(2);
     const subregion = composeSubregion(loadSpec.subregion, maxExtent);
 
     const levelIdx = pickLevelToLoad({ ...loadSpec, subregion }, this.getLevelShapesZYX());
-    const level = this.scaleLevels[levelIdx];
-    const levelShape = level.shape;
+    const arraysAtLevel = this.sources.map((src) => src.scaleLevels[levelIdx]);
```

Comment:
maybe this is just `level = this.sources[0].scaleLevels[levelIdx]; levelShape=level.shape` without having to map a new array ?

Reply:
`arraysAtLevel` is passed into `beginPrefetch` way down lower in this function, though it could also just receive the scale level index

Reply:
The thing I described above is nicer, so I switched to that